wpsc-includes/country-api.php

```diff
@@ -0,0 +1,193 @@
+<?php
```

Comment:
@JustinSainton @leewillis77 I would like this file to contain our outward facing procedural api for countries/regions/currency operations.  dev's certainly can go right at the class interface for the WPSC_Countries class, but that has a fair amount of complexisty because of some backwards compatibility requirements.

For most simple operations that a plugin or theme might want to do this file seem to be the API we shoulf promote, in combinatiuon with the WPSC_Country and WPSC_Region objects that the functions within the file return